testing/tests/DevExpress.ui.widgets/listParts/commonTests.js

```diff
@@ -4030,3 +4031,81 @@ if(devices.real().deviceType === 'desktop') {
         });
     });
 }
+
+if(QUnit.urlParams['nojquery']) {
+    QUnit.module('ShadowDOM', {
+    }, () => {
+        QUnit.test('ShadowDom: drag item', function(assert) {
+            const root = document.querySelector('#list');
+            const container = document.createElement('div');
+            const list = document.createElement('div');
+
+            root.attachShadow({ mode: 'open' });
+            container.appendChild(list);
+            root.shadowRoot.appendChild(container);
```

Comment:
let's move all duplicated code to the module `beforeEach`: shadow dom setup, items array, $items getting, pointer creating